### PR TITLE
Getting started locally overhaul

### PR DESCRIPTION
**Pull request by lukasbuenger**
Related to livingdocsIO/livingdocs-planning#2549.

I'd really like to give a little "further reading" to each of the two introductory paragraphs on each boilerplate. Any suggestions? Just point to the reference?

### CODE REVIEW

File: walkthroughs/getting-started-with-local-development.md [hunk 1]
```diff
@@ -1,86 +1,47 @@
...
+- [Docker](https://docs.docker.com/get-started/) - Very broadly, Docker is a tool designed to make it easier to create, deploy, and run applications by using containers. We use it to run our storage services (databases, search index and so on) on your local machine.
+- [GraphicsMagick](http://www.graphicsmagick.org/README.html) - A library that, among other things, enables image processing. We use it mainly to resize images.
+- [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) - Everybody's favorite distributed version control software. We use it extensively and, let's be honest, chances are you are too.
+- [jq](https://stedolan.github.io/jq/download/) - A lightweight and flexible command-line JSON processor. We use it for batch patches and data migrations.
```
Comment (gabrielhase) [inline walkthroughs/getting-started-with-local-development.md]: the setting of the environment to `local` is not in here anymore. Isn't it necessary anymore?

File: walkthroughs/getting-started-with-local-development.md [hunk 2]
```diff
@@ -1,86 +1,47 @@
...
-For example, you will want a different database configuration for production than when running Livingdocs locally.   
-
-## Config structure
-
-In the configuration folder, you find a global configuration in `environments/all`, that you can override with
-environment-specific values in `environments/{environment}.coffee`. These files are checked in to source control and therefore shared with everyone working with the repository.
+Now, what did you get?
 
-For local overrides you can use `secrets/{environment}.coffee` in the server and `user_specific/{environment}.coffee` in the editor. These files are not checked in to source control and therefore specific to your local installation.
+Within `livingdocs-server-boilerplate` we have a default configuration for a Livingdocs Server instance. This server, once up and running, will take care of a variety of things, from persisting your documents to search indexation – however, it is exclusively data driven or [headless](https://en.wikipedia.org/wiki/Headless_software).
```
Comment (gabrielhase) [inline walkthroughs/getting-started-with-local-development.md]: I would mention that it is a default local environment and only works for local development, i.e. when the ENV is set to `local`. I would also mention that it includes rate limited third-party accounts that we provide for free for evaluations.

File: walkthroughs/getting-started-with-local-development.md [hunk 3]
```diff
@@ -1,86 +1,47 @@
...
 
-## Sensitive configuration
+We still expect beautiful things to happen in our browser in the near future though. This is what `livingdocs-editor-boilerplate` is for. It contains a default configuration for a Livingdocs Editor instance, our editing user interface.
```
Comment (gabrielhase) [inline walkthroughs/getting-started-with-local-development.md]: I would refrain from poesie in a documentation ;) just write that there is also an editor configuration.

File: walkthroughs/getting-started-with-local-development.md [hunk 4]
```diff
@@ -1,86 +1,47 @@
...
-```
+- [livingdocs-editor-boilerplate](https://github.com/livingdocsIO/livingdocs-editor-boilerplate)
+- [livingdocs-server-boilerplate](https://github.com/livingdocsIO/livingdocs-server-boilerplate)
 
-## Setting the environment
+#### Persist ENVIRONMENT variable
 
-The environment derives from the environment variable `ENVIRONMENT`.
-For example to set the environment to local, run
+Locally, you might want to permanently set `ENVIRONMENT=local` by adding the following line to your `~/.zhsrc` or `~/.bashrc`
```
Comment (gabrielhase) [inline walkthroughs/getting-started-with-local-development.md]: ah, here it is :)

File: walkthroughs/getting-started-with-local-development.md (hunk 1 above)
Comment (lukasbuenger) [reply walkthroughs/getting-started-with-local-development.md]: The environment hint reappears throughout both boilerplate README's, so the info is not lost. It seemed a bit redundant even.

**Pull request merged**